.github/ISSUE_TEMPLATE/bug_report_zh.yml

```
@@ -0,0 +1,52 @@
+      description: 也请告诉我们，您期望发生什么？
+      placeholder: 告诉我们您看到了什么！
+      value: "出现了一个 bug！"
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: 重现步骤
+      description: "您如何触发此错误？请一步一步地引导我们。"
+      placeholder: |
+        1. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)